src/hip/hip_build_utils.cpp

```diff
@@ -96,7 +96,7 @@ boost::filesystem::path HipBuild(boost::optional<TmpDir>& tmp_dir,
     WriteFile(src, tmp_dir->path / filename);
     if(IsHccCompiler())
     {
-        params += " -amdgpu-target=" + dev_name;
+        params += " --std=c++11 -amdgpu-target=" + dev_name;
```

Comment:
Is this change really needed? This can prevent using newer c++'s. The kernel should pass the c++ version as part of the build flags.

Reply:
Reverted.